Busybox has a restriction in temp file name. From documentation:
Create a temporary file with name based on TEMPLATE and print its name.
TEMPLATE must end with XXXXXX (e.g. [/dir/]nameXXXXXX).

Use 6 x 'X' to be busybox compatible too.

Signed-off-by: Andrei Gherzan <andrei@gherzan.ro>

Index: git/omxplayer
===================================================================
--- git.orig/omxplayer
+++ git/omxplayer
@@ -43,8 +43,8 @@ if [ -z $NOREFRESH ] || [ "$NOREFRESH" =
     fi
 fi
 
-OMXPLAYER_DBUS_ADDR=`mktemp -t omxplayer-XXXXX`
-OMXPLAYER_DBUS_PID=`mktemp -t omxplayer-XXXXX`
+OMXPLAYER_DBUS_ADDR=`mktemp -t omxplayer-XXXXXX`
+OMXPLAYER_DBUS_PID=`mktemp -t omxplayer-XXXXXX`
 
 exec 5> $OMXPLAYER_DBUS_ADDR
 exec 6> $OMXPLAYER_DBUS_PID
